doc/api/stream.md

```diff
@@ -120,6 +119,12 @@ writing data *to* the socket. Because data may be written to the socket at a
 faster or slower rate than data is received, each side should
 operate (and buffer) independently of the other.
 
+The mechanics of the internal buffering are considered an internal
+implementation detail and may be changed at any time. However, for certain
+advanced implementations, the internal buffers can be retrieved using
+`writable.writableBuffer` or `readable.readableBuffer`, respectively. Use of
+these undocumented properties is not recommended.
```

Comment:
There's no ordering that "respectively" is obviously referring to here, and the names make it clear what they are.

```suggestion
`writable.writableBuffer` or `readable.readableBuffer`. Use of
```